hiring/hiring-process/engineering-hiring/scoring-a-take-home-exercise.md

```diff
@@ -1,53 +1,58 @@
-|  |  |  |
-| **Minimum Score Required** | **4** | **5** |
-
-## Pair Review
-
-> TO BE DONE
+We score a solution on seven criteria, which have been selected to take into account the fact that candidates might have different preferences for libraries/frameworks than us because of circumstances, and we look for general programming ability rather than familiarity with specific tools. 
+
+Every submitted exercise will be graded based on the [rubric](scoring-a-take-home-exercise.md#scoring-rubric) and will be given a score (from **0** to **14** points). We have a certain cut-off score, which an exercise will have to meet to decide if a candidate should be taken forward to the next stage.
+
+If a solution doesn't meet our standards, we **will** respond with an appropriate rejection mail.
+
+## Scoring rubric
+
+The scoring has seven different sections \(see table below\), all of which contribute a certain number of points to the overall score of a solution. The scoring process has been designed to compensate for a solution lacking in one aspect, but done well in others.
+
+1. For each of the sections in the table below, look at the state of the repository and select an appropriate score.
+2. Sum up the scores for the individual sections.
+3. If the total score is greater than or equal to the cut-off score, move the candidate to the next stage of the interview.
+4. If the total score is less than the cut-off score, respond with an email to the candidate which contains the score and what criteria they lost points in.
+
+| Criteria                                                                                                                                                                                                                         | Score |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------|
+| **Application**                                                                                                                                                                                                                  |       |
+| Barebones, developer UI with no error/touch feedback to the user.                                                                                                                                                                | 0     |
+| Functional UI, and with error feedback and/or touch feedback.                                                                                                                                                                    | 1     |
+| Polished UI, with thought put into the user experience and/or with nice transitions and animations.                                                                                                                              | 2     |
```

Comment:
The THE says that you won't be scored on the UI of the app.
This seems conflicting.

Reply:
That's a good point. We'll update the THE to remove that line and maybe link to this page on the exercise.

Reply:
Will update the THE docs once the PR is merged.